packages/common/src/schemas/json/lightdash-0.1.json

```diff
@@ -0,0 +1,190 @@
+                    "docs": {
+                        "type": "object",
+                        "properties": {
+                            "show": {
+                                "type": "boolean"
+                            }
+                        }
+                    },
+                    "tests": {
+                        "type": "array",
+                        "items": {
+                            "type": ["string", "object"]
+                        }
+                    },
+                    "meta": {
+                        "type": "object",
+                        "properties": {
+                            "joins": {
+                                "type": "array",
+                                "items": {
+                                    "type": "object",
+                                    "properties": {
+                                        "join": {
+                                            "type": "string"
+                                        },
+                                        "sql_on": {
+                                            "type": "string"
+                                        }
+                                    },
+                                    "required": ["join", "sql_on"]
+                                }
+                            }
+                        }
+                    },
+                    "columns": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "type": "string"
+                                },
+                                "description": {
+                                    "type": "string"
+                                },
+                                "quote": {
+                                    "type": "boolean"
+                                },
+                                "tests": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": ["string", "object"]
+                                    }
+                                },
+                                "tags": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "string"
+                                    }
+                                },
+                                "meta": {
+                                    "type": "object",
+                                    "properties": {
+                                        "metrics": {
+                                            "type": "object",
+                                            "patternProperties": {
+                                                "^[a-z0-9_]+$": {
```

Comment:
Should we support uppercase letters ?

Reply:
do we support uppercase letters for field names? @ZeRego

Reply:
decided to leave this as lower case only